default.nix

```diff
@@ -34,6 +34,7 @@ let
     );
   throwSystem = throw "Unsupported system: ${pkgs.stdenv.hostPlatform.system}";
   packages = self: {
+    temp = cardanoSL;
```

Comment:
why's this variable called temp?

Reply:
mainly to let the external file access cardano, but i will merge it into here